web/docs/understand/query-language/operators/head.md

```diff
@@ -0,0 +1,17 @@
+# head
+
+Limits the input to the first N results. The limit defaults to 1.
```

Comment:
```suggestion
Limits the input to the first N results. The limit defaults to 10.
```

This is in line with splunk and most other tools and what users expect.

Reply:
I changed the default to 10 for both, and also updated the documentation accordingly.

Reply:
Great, thanks!